bylaws/009-funding.md

```diff
@@ -0,0 +1,47 @@
+# Funding
+
+## Scope and objectives
+
+PHP-FIG is, as it often happens in Open Source communities, an organization composed by volunteers only; this doesn't exclude the fact that the organization itself may have some costs to sustain itself, like for maintaining a couple of tools such as a site and an email account. For such and similar reasons, we need a way to raise and govern money inside our organization.
+
+This document defines the roles, rules and processes that must be followed inside the PHP-FIG in all matters regarding money, to guarantee fairness, transparency and impartiality. This document intentionally describes the only allowed processes and ways to raise, administer and spend money, so that any change to such processes or any variation in budget allocation requires a bylaw change vote. 
+
+## Fiscal hosting
```

Comment:
```suggestion
## Fiscal hosting

```

I find it much easier to read markdown files if there's an extra blank line between the titles and body.  (Same elsewhere in the file.)